spack-repo/packages/singularity-eos/package.py

```diff
@@ -183,6 +214,15 @@ def cmake_args(self):
             self.define("SINGULARITY_USE_EOSPAC", "^eospac" in self.spec),
         ]
 
+        if "none" not in self.spec.variants["plugins"].value:
+            pdirs = []
+            for p in self.spec.variants["plugins"].value:
+                pdirs.append(join_path(self.stage.source_path, self.plugins[p]))
+            args.append(self.define("SINGULARITY_PLUGINS", ";".join(pdirs)))
```

Comment:
Did not know about `string.join`. Neat!

Reply:
This comment made me realize I can make a list comprehension above.